packages/sdk/src/types/index.ts

```diff
@@ -1,11 +1,12 @@
+export * from "./ConnectorError";
+export * from "./ConnectorResponse";
 export * from "./account";
 export * from "./attributes";
 export * from "./challenges";
-export * from "./ConnectorError";
-export * from "./ConnectorResponse";
 export * from "./files";
 export * from "./messages";
-export * from "./relationships";
 export * from "./relationshipTemplates";
+export * from "./relationships";
 export * from "./requests";
 export * from "./tokens";
+export type TODO = any;
```

Comment:
?? :D

Reply:
Helper type to mark a type as todo that is recognized by the todo plugin.

will move it to an file that is not expored by defautl

Reply:
just remove it pls. TODO comments shouldn't be in a PR